README.md

```diff
@@ -229,6 +229,57 @@ Flags:
 Use "in-toto record [command] --help" for more information about a command.
 ```
 
+### Completion
+
+```text
+Generate completion script
+Usage:
+  in-toto completion [bash|zsh|fish|powershell]
+
+Flags:
+  -h, --help   help for completion
+```
+
+#### Bash
+
+```shell
+$ source <(in-toto completion bash)
+# To load completions for each session, execute once:
+# Linux:
+$ in-toto completion bash > /etc/bash_completion.d/in-toto
```

Comment:
Hmm, on my machine, this is at `/usr/share/bash-completion`. Let me check some documentation for this.

Reply:
https://wiki.archlinux.org/title/bash#Tab_completion Looks like Arch's `bash-completion` package puts it there, and is sourced in bashrc. Other distros seem to do it the way you've listed here. I think it's okay to leave the destination as is but perhaps we can warn in the comment that it may differ from distro to distro?